R-package/DESCRIPTION

```diff
@@ -64,5 +63,6 @@ Imports:
     methods,
     data.table (>= 1.9.6),
     magrittr (>= 1.5),
+    jsonlite (>= 1.6),
```

Comment:
@hcho3 one more thing I forgot to comment on...do you know for sure that the code you've introduced is incompatible with older versions of `{jsonlite}`? Version 1.6 is almost 2 years old so this floor is reasonable, but if you don't have a specific reason for adding this floor I'd change this to `(>= 1.0)`

I think that it's better for libraries to only have floors / ceilings on major versions (like `>=1.0`) unless they know about a specific incompatibility with some minor version. That makes your package work for the largest number of possible users.

Reply:
Given that LightGBM is using 1.0+, it should be fine for XGBoost to use it too.